src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -586,6 +608,13 @@ export default {
 				this.$refs.actions.$refs.menuButton.$el.blur()
 			}
 		},
+
+		/**
+		 * Is this an external link
```

Comment:
External means not from the same domain, no ?
Here it looks like you just check if it has a protocol. Also, there is probably some domains that starts with `'http'`

Reply:
Also, explicitly check if `href` is not `null`: `href !== null`.

Reply:
Why not check against invalid href (e.g. empty strings?)

Reply:
> External means not from the same domain, no ?

Well, yes and no, a nextcloud instance can have multiple domains.
That's why I checked for http only :)

Reply:
I double-checked, and it should not cause an issue here, but I am always afraid of implicit comparisons. :fearful:  We have some of them in the backend that caused issues. Maybe in the front-end too, but can't remember a specific example. I usually tend to avoid them as it reduce easy mistake, and cognitive load.

![image](https://user-images.githubusercontent.com/6653109/210740659-dc64c602-644b-4aa7-be72-794c23e49f24.png)

Reply:
> Well, yes and no, a nextcloud instance can have multiple domains.

Indeed

If it is only a protocol check, then I would call it that way and use a regexp to check for `'://'`. Something around `/https?:\/\//` ?

Reply:
> I double-checked, and it should not cause an issue here, but I am always afraid of implicit comparisons.

This is not a comparison, this is a falsy check. 
![image](https://user-images.githubusercontent.com/14975046/210746564-fd48f428-a158-4abf-98ea-e1fc77366119.png)